feat: create FAQ documentation (#393)
ublue-os/website: docs/images/bazzite/FAQ.md

* [Heroic](https://flathub.org/apps/com.heroicgameslauncher.hgl) for Epic, GOG, and Amazon Games Launcher 

# How do I run Android applications?

We have a guide on that [here.](https://universal-blue.org/images/bazzite/waydroid/)

# How do updates work?

### **For *bazzite*, *bazzite-nvidia*, *bazzite-gnome*, and *bazzite-gnome-nvidia*:**

System updates happen automatically daily.  They will be downloaded in the background and will apply on shutdown.  No forced reboots or worrying about manually updating your system.

Flatpak applications (installed from *Discover* or *GNOME Software*) update twice a day automatically.

You can however force update to the whole system (base packages, applications, and containers) at any time by opening your host terminal and entering `just update` then reboot your device after it has finished.

### **For *bazzite-deck* and *bazzite-deck-gnome*:**

Similar to SteamOS, updates are handled by Steam.  In Game Mode, go to Settings > System > click "Apply"

Alternatively, you can open a host terminal and enter `just update` then reboot your device after it has finished.

# How do I install additional software?

### Flatpak

> Typically it is recommended to use Flatpak for most software.  These can be installed via the software center that is preinstalled like *Discover* or *GNOME Software*.  Take a look at the [selection.](https://flathub.org/apps/collection/popular/1)

Reviewer: ```suggestion
Typically it is recommended to use Flatpak for most software.  These can be installed via the software center that is preinstalled like *Discover* or *GNOME Software*.  Take a look at the [selection](https://flathub.org/apps/collection/popular/1).
```

### AppImage

> AppImages can be installed by downloading any file with an `.AppImage` extension.  These are usually found on a project's website, [Appimage Hub](https://www.appimagehub.com/) or if you install [Appimage Pool.](https://flathub.org/apps/io.github.prateekmedia.appimagepool)  Then giving the AppImage application executable permssions in the file's properties to run.

Reviewer: ```suggestion
AppImages can be installed by downloading any file with an `.AppImage` extension.  These are usually found on a project's website, [Appimage Hub](https://www.appimagehub.com/) or if you install [Appimage Pool](https://flathub.org/apps/io.github.prateekmedia.appimagepool). Then giving the AppImage application executable permssions in the file's properties to run.

Note that AppImages are not supported in other Universal Blue images.
```

### Distrobox

> Distrobox containers can be made with the host terminal following this [documentation.](https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-create.md)  Keep in mind that `bazzite-arch` is a distrobox container that uses pacman as the package manager and can utilize the [AUR.](https://aur.archlinux.org/)  Take a look at other distros you can run inside of a container [here.](https://github.com/89luca89/distrobox/blob/main/docs/compatibility.md#containers-distros)

Reviewer: ```suggestion
Distrobox containers can be made with the host terminal following this [documentation](https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-create.md). Keep in mind that `bazzite-arch` is a distrobox container that uses pacman as the package manager and can utilize the [AUR](https://aur.archlinux.org/).

You can use [other distributions](https://github.com/89luca89/distrobox/blob/main/docs/compatibility.md#containers-distros) inside the container.
```

### Nix

If you opted to use Nix packages at the first boot, then they can be installed by entering:  
`nix-env --install <package>` in a host terminal.  Click [here](https://search.nixos.org/packages) to search for Nix packages.

### rpm-ostree

If you need something to be at a "host level" like an obscure VPN client or virt-manager. Open a host terminal and enter `rpm-ostree install <package>` and **reboot** the device after it has finished.  These packages are Fedora packages that substitute both `dnf` and `yum` on a tradtional Fedora Linux system.  

> **NOTE:** This may take a while since it's not only installing the package(s) you entered, but also deploying a new image of the OS with these packages overlayed onto it. 

Reviewer: Just link people to fedora's rpm-ostree manual here, this is too incomplete.

# Do I have to reboot after every `system` update or layering a package?

No.  They just won't apply until shutdown.  You can attempt to layer package(s) without rebooting with `rpm-ostree install --apply-live <package>` but sometimes this still requires a reboot depending on the package(s) you installed.

# Why use Fedora? SteamOS is based on Arch Linux.

SteamOS is based on Arch Linux, but the base packages and drivers get updates at a much slower pace than using vanilla Arch and updating yourself.  Bazzite will follow Fedora's updates which means it will always be ahead of SteamOS in terms of newer software and drivers.

> # Is Bazzite "immutable?"
> 
> Yes.  Like SteamOS, there are read-only root files for stability purposes.
> 
> The immutable desktop version of Fedora is built on [libostree](https://ostreedev.github.io/ostree/) which has advantages such as:
> * Atomic upgrades.
> * Containerized applications. (**No dependency hell!**)
> * Overlay system packages to the host.
> * Smoother upgrade process from major Fedora point releases.
> * Very little risk of an unbootable or broken system.
> * Rollback system updates.

Reviewer: ```suggestion
# How does Bazzite differ from other Linux distributions?

Yes.  Like SteamOS, there are read-only root files for stability purposes.

Bazzite is built off of the desktop versions of Fedora is built [libostree](https://ostreedev.github.io/ostree/) which has advantages such as:
* Atomic upgrades.
* Containerized applications. (**No dependency hell!**)
* Overlay system packages to the host.
* Smoother upgrade process from major Fedora point releases.
* Very little risk of an unbootable or broken system.
* Rollback system updates.

Check out the [Universal Blue homepage](https://universal-blue.org) for more information. 
```

We have all this on the homepage. We don't use the term immutable here.

# Is this another fringe distro?

> This is Fedora Kinoite/Silverblue, depending on the desktop environment you choose, with a recipe on top of it.  Unlike traditional Linux distributions, much of the maintenance and security updates are done upstream by Fedora and Universal Blue while Bazzzite only has to worry about making this a great gaming experience out of the box.

Reviewer: ```suggestion
This is Fedora Kinoite/Silverblue, depending on the desktop environment you choose, with a recipe on top of it.  

Unlike traditional Linux distributions, much of the maintenance and security updates are done upstream by Fedora and Universal Blue while Bazzite only configures a great gaming experience out of the box. Check the [mission statement](/mission) for more information. 
```
